src/components/Sidenav/SidenavDocsLogo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { theme } from '../../theme/docsTheme';
+import styled from '@emotion/styled';
```

Comment:
(Nit) Library imports on top for consistency

```suggestion
import styled from '@emotion/styled';
import { theme } from '../../theme/docsTheme';
```